proxy/proxy.go

```diff
@@ -92,7 +92,10 @@ func (s *Settings) AsEnvironmentValues() []string {
 // DefaultEnvironment="http_proxy=...." "HTTP_PROXY=..." ...
 func (s *Settings) AsSystemdDefaultEnv() string {
 	lines := s.AsEnvironmentValues()
-	rv := `[Manager]
+	rv := `# To make juju control global systemd proxy settings create
```

Comment:
I'm finding this a bit hard to parse.

Perhaps:

    # To allow juju to control the global systemd proxy settings,
    # create symbolic links to this file from within /etc/systemd/system.conf.d/
    # and /etc/systemd/users.conf.d/.


?